404.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>Notfound.org</title>
+</head>
+
+<body>
+  <iframe src="http://notfound-static.fwebservices.be/404/index.html?&amp;key=d1acaa1dbbbb196ec2c3ced174d4fd41" width="100%"
```

Comment:
do we want to display this or do we want to forward to one of our actual 404 web pages?